src/Designer/backend/src/Designer/wwwroot/css/startpage.css

```
@@ -19,7 +19,7 @@
 }
 
 * {
-    font-family: DIN-reg;
+    font-family: 'Inter', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)